infrastructure/src/main/java/org/wso2/testgrid/infrastructure/providers/KubernetesProvider.java

```diff
@@ -0,0 +1,221 @@
+                    + " --input-dir " + testInputsLoc;
+            int exitCode = executor.executeCommand(command);
+            return exitCode == 0;
+        } catch (CommandExecutionException e) {
+            throw new TestGridInfrastructureException(
+                    "Exception occurred while executing the infra-destroy script " + "for deployment-pattern '"
+                            + infrastructureConfig.getFirstProvisioner().getName() + "'", e);
+        }
+    }
+
+    /**
+     * This is used to get the access key to log into the GKE
+     *
+     * @return the access key location
+     */
+
+    private String getAccessKeyFileLocation(){
+        String accessKeyFileLocation=null;
+        try{
+            accessKeyFileLocation=ConfigurationContext.getProperty(ConfigurationContext
+                    .ConfigurationProperties.GKE_ACCESS_KEY);
+            logger.info(accessKeyFileLocation);
+                   }catch(PropertyNotFoundException e){
+            logger.error("The keyFileLocation is not found");
+        }
+        return accessKeyFileLocation;
+    }
+
+    /**
+     * This is used to set the access key location
+     *
+     * @param testPlan with current test run specifications
+     */
+
+    private void setAccessKeyFileLocation(TestPlan testPlan) {
+        final Path location = DataBucketsHelper.getInputLocation(testPlan)
+                .resolve(DataBucketsHelper.TESTPLAN_PROPERTIES_FILE);
+        DeploymentConfig.DeploymentPattern deploymentPatternConfig = testPlan.getDeploymentConfig()
+                .getDeploymentPatterns().get(0);
+        final String accessKeyFileLocation = getAccessKeyFileLocation();
+        try (OutputStream os = Files.newOutputStream(location, CREATE, APPEND)) {
+            os.write(("\nname="+deploymentPatternConfig.getName()).getBytes(StandardCharsets.UTF_8));
+            os.write(("\n" +TestGridConstants.ACCESS_KEY_FILE_LOCATION + "=" +
+                    accessKeyFileLocation).getBytes(StandardCharsets.UTF_8));
+        } catch (IOException e) {
+            logger.error("Error while persisting infra input params to " + location, e);
+        }
+    }
+
+    /**
+     * This sets the properties to be used by the deploy scripts.
+     *
+     * @param testPlan
+     */
+
+    private void setProperties(TestPlan testPlan) {
+        String WUM_USERNAME = null;
+        String WUM_PASSWORD = null;
+        final Path location = DataBucketsHelper.getInputLocation(testPlan)
+                .resolve(DataBucketsHelper.INFRA_OUT_FILE);
+        logger.info(location.toString());
+        try{
+            WUM_USERNAME=ConfigurationContext.getProperty(ConfigurationContext.
+                    ConfigurationProperties.WUM_USERNAME);
+            WUM_PASSWORD=ConfigurationContext.getProperty(ConfigurationContext.
+                    ConfigurationProperties.WUM_PASSWORD);
+        }catch(PropertyNotFoundException e){
```

Comment:
Could you please run `mvn clean install` from your command-line from the project root? That'll execute findbugs and checkstyle plugin which'll alert you on formatting issues.

Reply:
okay I will